modules/hjt/hjt.go

```diff
@@ -51,7 +51,7 @@ func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, cmd string,
 	if len(values) == 0 {
 		_, err = ds.ChannelMessageSend(mc.ChannelID, "No matches found")
 	} else {
-		_, err = ds.ChannelMessageSend(mc.ChannelID, strings.Join(values, ", "))
+		_, err = ds.ChannelMessageSend(mc.ChannelID, strings.Join(values, ",\n"))
```

Comment:
Now instead of

> MCLeaks/AltDispenser(Tool, Pirated, Fix: info/alt then win/hosts/edit), BitDefender(AV, Bad, Info:av/bitdefender), Overwolf(Old versions crash, Fix: Update Overwolf), Discord(Turn off Discord Overlay, else disable Discord), VPN(Any VPN), MCLeaks/AltDispenser(Tool, Pirated, Fix: info/alt then win/hosts/edit)

it will return 

> MCLeaks/AltDispenser(Tool, Pirated, Fix: info/alt then win/hosts/edit),
> BitDefender(AV, Bad, Info:av/bitdefender),
> Overwolf(Old versions crash, Fix: Update Overwolf),
> Discord(Turn off Discord Overlay, else disable Discord),
> VPN(Any VPN),
> MCLeaks/AltDispenser(Tool, Pirated, Fix: info/alt then win/hosts/edit)